website/src/css/custom.css

```diff
@@ -107,3 +107,8 @@ svg.lightToggleIcon_node_modules-\@docusaurus-theme-classic-lib-next-theme-Color
 li.footer__item a svg {
   display: inline;
 }
+
```

Comment:
We should not customize it there

Reply:
Where would be the best way to customize? looking through tailwindcss docs I found that you can use `[data-theme='dark']` to set it in a custom.css

Still learning, so I apologize!

Reply:
I think we're following the 'we set the colors where we need it' rule

so if you want to use one color for the text, you apply 'text-theme-level' for white theme and 'dark:text-theme-level' for dark theme.

so global override, it's a per element customization

Reply:
Got it, so each `p`, I'll add a `dark:text-gray-100` instead. Although it will be more line changes, at least it's the tailwindcss way.

Reply:
what you can do, is a new component like <Subtitle> and inside you apply dark:text-gray-100 and everytime you want a "subtitle" you reuse that component instead of <p>

Reply:
leveraging that there is react or svelte under the hood, so bring new component/helpers